multimodal/src/autogluon/multimodal/learners/base.py

```diff
@@ -1131,8 +1143,14 @@ def init_trainer_per_run(
             log_filter = LogFilter(blacklist_msgs)
 
             with apply_log_filter(log_filter):
+                if num_gpus > 0:
+                    accelerator = "gpu"
+                elif OmegaConf is not None:
```

Comment:
When is OmegaConf None?

Reply:
It was a typo, should be config. The cpu only error is fixed. But t-few (or AutoModelForSeq2SeqLM) still have a problem using DDPCacheWriter. Because `target_template_logits` is not using `batch_size` but `batch_size*num_choices` as first dimension. I'm still working on the fix.